h2o-app/build.gradle

```diff
@@ -9,6 +9,7 @@ dependencies {
   compile project(":h2o-algos")
   compile project(":h2o-automl")
   compile project(":h2o-ext-target-encoder")
+  compile project(":h2o-k8s")
```

Comment:
Should this really be added to h2o-app? Or should it only go to the main assembly? I would expect to see it only in assembly.

Reply:
Oh snap ! That's forgotten. It's part of the assembly of course. Removed in https://github.com/h2oai/h2o-3/pull/4554/commits/785505a19fea1cb894fc02ca48ece25836d8c062.